src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -212,7 +212,7 @@ private string GetLink(SourceInfo<string> href)
 
             var (error, link, display, _) = href.HasValue
                 ? _xrefResolver.ResolveXrefByHref(href.Value, (Document)InclusionContext.File, (Document)InclusionContext.RootFile)
-                : _xrefResolver.ResolveXrefByUid(uid ?? throw new InvalidOperationException("href and uid can't be both null"), (Document)InclusionContext.File, (Document)InclusionContext.RootFile);
+                : _xrefResolver.ResolveXrefByUid(uid ?? new SourceInfo<string>(""), (Document)InclusionContext.File, (Document)InclusionContext.RootFile);
```

Comment:
consider to refine the method signature to `SourceInfo<string>?`, and return early when null